src/core/ProgressBar/BaseProgressBar/styles.ts

```diff
@@ -0,0 +1,30 @@
+import { makeStyles, Theme } from '@material-ui/core';
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    width: '12.125rem',
+    borderRadius: '0.125rem',
+  },
+  label: {
+    fontSize: '0.75rem',
+    lineHeight: '0.875rem',
+    marginBottom: theme.spacing(1.625),
+    display: 'block',
+    color: theme.palette.text.primary,
```

Comment:
Use this from `props` of useStyle

Reply:
@S-ayanide I'm not sure what should be done regarding the props, in figma we have the text text primary, could you clarify pls

Reply:
You would be sending `color` as a props to useStyle, as already mentioned above like `useStyle({ color })` and use the color here as `props.color`